### PR TITLE
Adds the 'CLI for Microsoft 365 v6' post

### PR DESCRIPTION
**Pull request by waldekmastykarz**
## Category

- [ ] Content fix
- [x] New article

> *(DELETE THIS PARAGRAPH AFTER READING)*

## Contents of the Pull Request

CLI for Microsoft 365 v6 announcement post

### CODE REVIEW

**Review by martinlingstuyl — APPROVED** [review]
Looks great @waldekmastykarz, no comments 👏

**Review by milanholemans — APPROVED** [review]
Looking good, one small remark.

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v6-0/index.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+m365 spfx project upgrade --toVersion 1.16.0 --output tour
+```
+
+We've also updated other SPFx-related commands to support SPFx v1.16.0.
+
+### Extended documentation with commands' output
+
+As more and more people use CLI for Microsoft 365 in scripts, we realized that we can improve the experience, by including in the documentation sample output of each command. This way, you can easily see what to expect from the command and how to parse the output, without having to run the command yourself first.
+
+![Example of the response of the planner plan list command in CLI for Microsoft 365](./images/response.png)
```
Comment (Jwaegebaert) [inline content/cli-for-microsoft-365/cli-for-microsoft-365-v6-0/index.md:65]: I can't seem to get the images rendered. Could be something on my side of course but worth a double check 😄

**Review by Jwaegebaert — APPROVED** [review]
Great job, the post looks awesome. I've one remark that could be worth taking a look at 🚀

File: content/cli-for-microsoft-365/cli-for-microsoft-365-v6-0/index.md (hunk 1 above)
Comment (waldekmastykarz) [reply content/cli-for-microsoft-365/cli-for-microsoft-365-v6-0/index.md:65]: Hmm, seems they're broken for some reason. Thanks, I'll have a look

**Pull request merged**